doc/sql.extensions/README.set_transaction.txt

```diff
@@ -27,6 +27,12 @@ commit fails. This option is mostly used by gfix.
 LOCK TIMEOUT nonneg_short_integer: it's the time (measured in seconds) that a
 transaction waits for a lock in a record before giving up and reporting an error.
 
+AUTO RELEASE TEMP BLOBID: makes the transaction release temporary ID of user BLOB
+just after its materialization. It's useful for massive insertions of records with
+user-defined BLOBs because it eliminates the memory overhead caused by creating and
+keeping temporary IDs until the transaction ends. This option is used during the
+database restore.
+
```

Comment:
I would add something like: use with care and only if there is no need to open just inserted (or updated) blob using temporary blob ID obtained after blob creation.